client/components/HateMailComponents/HateMail.jsx

```diff
@@ -97,7 +117,10 @@ const HateMail = ({ user }) => {
               required
             />
           </div>
-          <button type="submit">Send Hatemail</button>
+          <button type="submit" onClick={() => {
+            postConflict()
+            console.log('src', recipients)
```

Comment:
There's a few leftover logs in this file and look like some unused code between 48-51

Reply:
Thank you, they've been removed.